2wp-app simplify the documentation (#929)
rsksmart/devportal: content/rsk-devportal/guides/two-way-peg-app/pegout/ledger.md

![2 way peg app (peg-out)](/assets/img/guides/two-way-peg-app/pegout.gif)

## Performing a peg-out transaction using rLogin(Trezor and Ledger)

> ​> - Note that we will be using the 2 way peg app on [2 way peg app - Testnet](https://2wp-app.testnet.rootstock.io/) for learning purposes.

Reviewer: ```suggestion
​> - Note that we will be using the 2 way peg app on [2 way peg app - Testnet](https://app.2wp.testnet.rootstock.io/) for learning purposes.
```

> > - For transactions using **real tokens**, use the [2 way peg app - Mainnet](https://2wp-app.rootstock.io/) application.

Reviewer: ```suggestion
> - For transactions using **real tokens**, use the [2 way peg app - Mainnet](https://app.2wp.rootstock.io/) application.
```

Reviewer 2: @alexjavabraz I'm going to push another commit, the docs aren't showing because some pages don't have metadata

Reviewer 2: @alexjavabraz Proofreading done, commit pushed and files organized. Please let me know when you incorporate @josedahlquist review. Thank you.

---
menu_order: 1400
menu_title: Peg-out using Ledger
title: "Performing a peg-out using Ledger Hardware Wallet | 2 way peg app Documentation"
description: "Here, we will learn how to perform a peg-out using Ledger Hardware Wallet."
tags: 2 way peg, powpeg, peg-in, peg-out, 2way-peg, bridge, rbtc, btc, testnet, mainnet, trezor, liquality, leger, guide, setup, integrate, use
layout: rsk
---

![2 way peg app (peg-out)](/assets/img/guides/two-way-peg-app/pegout.gif)

## Performing a peg-out transaction using rLogin(Trezor and Ledger)

​> - Note that we will be using the 2 way peg app on [2 way peg app - Testnet](https://app.2wp.testnet.rootstock.io/) for learning purposes.
> - For transactions using **real tokens**, use the [2 way peg app - Mainnet](https://app.2wp.rootstock.io/) application.
> - We're using Ledger Nano and Trezor One hardware wallets on this tutorial.

> > - For how to perform a peg-in transaction using hardware wallets. See [Getting started using hardware wallets](/guides/two-way-peg-app/getting-started#using-hardware-wallets).
> > - To use Ledger hardware wallet to create a **peg-in** see [How to perform a peg-in transaction using Ledger](/guides/two-way-peg-app/getting-started#performing-a-peg-in-transaction-with-ledger)
> > - To use Trezor hardware wallet to create a **peg-in** see [How to perform a peg-in transaction using Ledger](/guides/two-way-peg-app/getting-started#trezor-hardware-wallet)

Reviewer 3: ```suggestion
> - To use Ledger hardware wallet to create a **peg-in** see [How to perform a peg-in transaction using Ledger](/guides/two-way-peg-app/pegin/ledger-hardware-wallet)
> - To use Trezor hardware wallet to create a **peg-in** see [How to perform a peg-in transaction using Trezor](/guides/two-way-peg-app/pegin/trezor-hardware-wallet)
```
